R/wb_functions.R

```diff
@@ -806,3 +806,72 @@ wb_set_selected <- function(wb, sheet) {
 
   wb
 }
+
+#' dummy function to add a chart to an existing workbook
+#' currently only a barplot is possible
+#' @param wb a workbook
+#' @param sheet the sheet on which the graph will appear
+#' @param dims the dimensions where the sheet will appear
+#' @param graph mschart graph
+#' @examples
+#' if (requireNamespace("mschart")) {
+#' require(mschart)
+#'
+#' ### Scatter
+#' scatter <- ms_scatterchart(data = iris, x = "Sepal.Length", y = "Sepal.Width", group = "Species")
+#' scatter <- chart_settings(scatter, scatterstyle = "marker")
+#'
+#' wb <- wb_workbook() %>%
+#'  wb_add_worksheet() %>%
+#'  wb_add_mschart(dims = "F4:L20", graph = scatter)
+#' }
+#' @export
+wb_add_mschart <- function(
+  wb,
+  sheet = current_sheet(),
+  dims = "B2:H8",
+  graph
+) {
+
+  assert_workbook(wb)
+  assert_class(graph, "ms_chart")
+
+  requireNamespace("mschart")
+
+  sheetname <- wb$.__enclos_env__$private$get_sheet_name(sheet)
+
+  # format.ms_chart is not exported
+  out_xml <- mschart:::format.ms_chart(
```

Comment:
use format.ms_chart and depend on mschart >= 0.4 otherwise we'd need our own format implementation

Reply:
In a wonderful we would have a `from_mschart()` and a `to_mschart()`. Unfortunately we have to deal with time restrictions and my currently limited interest in writing these functions.